crates/zksync/core/src/vm/tracer.rs

```diff
@@ -56,6 +56,9 @@ const SELECTOR_SYSTEM_CONTEXT_BLOCK_NUMBER: [u8; 4] = hex!("42cbb15c");
 /// Selector for `getBlockTimestamp()`
 const SELECTOR_SYSTEM_CONTEXT_BLOCK_TIMESTAMP: [u8; 4] = hex!("796b89b9");
 
+/// Selector for `block.baseFee`
+const SELECTOR_BLOCK_BASE_FEE: [u8; 4] = hex!("6ef25c3a");
```

Comment:
The docs need to follow the above format. Also the selector is for `baseFee()` function